scripts/hadoop/Dockerfile

```diff
@@ -0,0 +1,77 @@
+FROM --platform=linux/amd64 fedora:40
```

Comment:
This, along with the generating the script in the dockerfile were changes I made when attempting to launch the hdfs instance from compose. I kept getting "EXEC format error" for the command when copying the script into the container and executing it via compose or start

Looking through the commit history, I think it might be due to an empty line above the shebang in `hdfs-init.sh`. I'll move the script back out